### PR TITLE
Add Planet Launcher

### LINKED ISSUE #1599: Planet Launcher
### What is the name of the app?

Planet Launcher

### (Optional) Where is the app hosted?

https://github.com/mcpiscript/planet

### About the app

It's a launcher that actually saves stuff and etc for mcpi. 

### Confirmations

- [X] I have confirmed that this app is legal and not piracy.
- [X] I have confirmed that this app has never been discussed in https://github.com/Botspot/pi-apps/issues and https://github.com/Botspot/pi-apps/pulls, and it is not in the Pi Apps app list.
- [x] I have confirmed that this app can run on Raspberry Pi.

### PR DESCRIPTION
**Pull request by leha-code**
Closes #1599

### CODE REVIEW

File: apps/Minecraft Pi (Modded)/install [hunk 1]
```diff
@@ -3,12 +3,18 @@
 
 # Remove old MCPILs
 sudo apt-get remove -y mcpil-r mcpil &>/dev/null || true
+
+# Install the repos
 wget -qO- https://raw.githubusercontent.com/MCPI-Revival/mcpi-packages/master/install.sh | grep -v 'sudo apt update' | sudo bash - || error "Failed to install APT repo!"
+curl -s --compressed "https://mcpiscript.github.io/PPA/KEY.gpg" | sudo apt-key add -
+sudo curl -s --compressed -o /etc/apt/sources.list.d/mcpiscript.list "https://mcpiscript.github.io/PPA/mcpiscript.list"
+sudo apt update
+
```
Comment (Botspot) [inline apps/Minecraft Pi (Modded)/install:12]: Are these lines only used if the Planet Launcher option is chosen? If so, please put them in an `if` statement.

File: apps/Minecraft Pi (Modded)/install [hunk 2]
```diff
@@ -3,12 +3,16 @@
 
 # Remove old MCPILs
 sudo apt-get remove -y mcpil-r mcpil &>/dev/null || true
+
+# Install the repos
 wget -qO- https://raw.githubusercontent.com/MCPI-Revival/mcpi-packages/master/install.sh | grep -v 'sudo apt update' | sudo bash - || error "Failed to install APT repo!"
```
[the remark this answers is not in the document]
Comment (leha-code) [reply apps/Minecraft Pi (Modded)/install:8]: This is mcpi-revival repo. It is necessary

**Comment by rbcoder117** [comment]
@Botspot  this is ready to be merged

**Comment by leha-code** [comment]
@Botspot

**Pull request merged**